rpc_util_test.go

```diff
@@ -290,3 +293,115 @@ func BenchmarkGZIPCompressor512KiB(b *testing.B) {
 func BenchmarkGZIPCompressor1MiB(b *testing.B) {
 	bmCompressor(b, 1024*1024, NewGZIPCompressor())
 }
+
+type MockCompressor struct {
```

Comment:
why "M" capital? We are not going to export this struct

Reply:
done